packages/manager/modules/cloud-connect/src/cloud-connect.controller.js

```diff
@@ -0,0 +1,73 @@
+import { map } from 'lodash';
+import { ListLayoutHelper } from '@ovh-ux/manager-ng-layout-helpers';
+import { STATUS } from './cloud-connect.constants';
+
+export default class CloudConnectCtrl extends ListLayoutHelper.ListLayoutCtrl {
+  /* @ngInject */
+  constructor($q, $http, $translate, ouiDatagridService, coreURLBuilder) {
+    super($q, ouiDatagridService);
+    this.$translate = $translate;
+    this.$http = $http;
+    this.coreURLBuilder = coreURLBuilder;
+  }
+
+  $onInit() {
+    this.datagridId = 'dg-cloudconnect';
+    this.defaultFilterColumn = 'uuid';
+
+    super.$onInit();
+
+    this.status = STATUS;
+    this.PROVIDER_OVH_CLOUD = 'OVHcloud';
+
+    this.columnsConfig = [
+      { name: 'uuid', sortable: this.getSorting('uuid') },
+      {
+        name: 'description',
+        sortable: this.getSorting('description'),
+      },
+      { name: 'pop', sortable: this.getSorting('pop') },
+      { name: 'vrack', sortable: this.getSorting('vrack') },
+      { name: 'status', sortable: this.getSorting('status') },
+    ];
+
+    this.orders = [];
+
+    // Check if there is order to follow up
+    this.getOrderFollowUp();
+  }
+
+  getOrderFollowUp() {
+    return this.$http.get('/ovhCloudConnect/order').then((res) => {
```

Comment:
could you use iceberg expand feature instead of making one xhr request per row ?

Reply:
@frenautvh Is there any documentation about how to implement iceberg expand for this update ? Thanks for your help.

Reply:
I found how to do do this.